js/select.js

```diff
@@ -6,77 +6,37 @@
     dropdownOptions: {}
   };
 
-  /**
-   * @class
-   *
-   */
   class FormSelect extends Component {
-    /**
-     * Construct FormSelect instance
-     * @constructor
-     * @param {Element} el
-     * @param {Object} options
-     */
     constructor(el, options) {
       super(FormSelect, el, options);
-
-      // Don't init if browser default version
-      if (this.$el.hasClass('browser-default')) {
-        return;
-      }
-
```

Comment:
hmm why did you remove most of the comments there?

Reply:
Because according to clean code, most of the comments were obsolete and had the same name as the functions, so i removed them to keep the code as clean as possible.

Reply:
haha Clean Code, as I expected 😅 

but additional enters / blank lines are fine IMO, no need to remove them (ironically you also add additional blank lines 😄)

so far reading the code LGTM, but I haven't tried it on one of my apps.

Reply:
wait.. uh oh... I just checked this and it somehow break my app 😅 

Though, it's kinda expected because I call the internal `_handleSelectChange` directly. Need some time to investigate this 🤕

Reply:
Yes thats correct, I also saw that on my Project. The text is not updated correctly. I will investigate again this week an provide an additional commit which fixes this issue. Thank you for taking some time and test it. Looking forward for improvements. Best greets!

Reply:
it should work now... i will test also in my project. You dont have to call the internal function but you can dispach a change Event: like `selectElem.dispatchEvent(new Event('change'))`

Reply:
Yes! It works now in my Project too 🖖

Reply:
I did another commit, because i now fixed an issue in the multi-select which was open.
I also improved the styling of the dropdown. It is now below the input. After this PR we can upgrade the Select with a Search function which would allow searching / ajax in the select and multi-select. The design change I made is also conform to the Material.io Specs from Google. The Dropdown is shown below the input. See this screenshot for details:
![grafik](https://user-images.githubusercontent.com/27806175/149948916-9414257d-a77e-4ef7-bbde-85c3bb3b487f.png)

Reply:
@Smankusors please check it in your project now. I tested it on my Project and it looks good now.
Looking forward for your response.

When should we do the next version iteration? Maybe 1.2.0 would be nice? What do you think @DanielRuf?
Then we could merge the other PRs too with breaking changes and then if it works for some time and things look good we could release a major 2.0.0.

Reply:
I think we can promote the current changes in the develop branch to `v1.1.0` since we only have an alpha release of it.

https://github.com/materializecss/materialize/commits/main?after=026fe0d666e7698a50192a34756fa4a3378d7cdf+69&branch=main

https://github.com/materializecss/materialize/tags

Can you coordinate this @Smankusors?
If you need some input from us, please let us know.

After this we can merge the current PRs and make a v1.2.0 release.

Reply:
> @Smankusors please check it in your project now. I tested it on my Project and it looks good now. Looking forward for your response.

oh thanks, sorry if this takes days. Running out of time on weekend 😩 

I will test it now

> Can you coordinate this @Smankusors? If you need some input from us, please let us know.
> 
> After this we can merge the current PRs and make a v2.0.0-rc release.

wait, coordinate what? To release `v2.0.0-rc`?

Reply:
Afaik most of these commens are useful for autogenerating documentation from the annotations.

> wait, coordinate what? To release v2.0.0-rc?

No, we are still at v1.1-alpha. With coordinate I mean "feel free to merge if these changes are fine and make a v1.1 (without alpha) release".

Reply:
> I think we can promote the current changes in the develop branch to v1.1.0 since we only have an alpha release of it.

That was what I meant with "coordinate".
You can make a PR for the v1.1 release so the knowledge for doing releases is shared.

Reply:
> Afaik most of these commens are useful for autogenerating documentation from the annotations.
> 
> > wait, coordinate what? To release v2.0.0-rc?
> 
> No, we are still at v1.1-alpha. With coordinate I mean "feel free to merge if these changes are fine and make a v1.1 (without alpha) release".

Yes but the names and descriptions where the same as the function names, so no advantage here. Also the documentation is handwritten for now so i thought i clean up the code and make it more compact. This can be merged from my point of view. Looking forward for some improvement of this project 👍